independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/alternatives/priority/AlternativePriorityResolutionTest.java

```diff
@@ -45,7 +45,7 @@ static class NoParentAlternativePriorityProducer1 {
         @Produces
         @Singleton
         @Alternative
-        @io.quarkus.arc.Priority(4)
+        @Priority(4)
```

Comment:
This test is now likely superfluous + the TCKs we are now running are covering this as well.

Reply:
Actually, this test might overlap with some others but it still probably valuable, let's keep it in.